### ISSUE TITLE
Calcium density profiles and concentrations in CHARMM36 and LIPID17 simulations

### ISSUE THREAD

**Issue by ohsOllila** [NMRLipids/NMRlipidsIVotherHGs#26]
I have made a new plot showing calcium densities only with lowest ion concentrations in POPC:POPS (5:1) systems: 
https://github.com/NMRLipids/NMRlipidsIVotherHGs/blob/master/Figs/CAdensPCPSmixtureLOWcons-eps-converted-to.pdf

There are following issues to be solved:
1) We have only electron densities from CHARMM36 simulations by Jesper Madsen, but we need the number densities for this plot.
2) The y-axis in the LIPID17 results by @batukav does not seem to be consistent with the mass nor electron densities from other simulations. These results should be checked. 
3) We need the information about the amount of ions in these simulations to calculate the correct concentrations. This applies also table II in the manuscript.

**Comment by batukav** [comment]
Hi,
Hopefully within one week I'll be able to complete issues 2) and 3).

**Comment by batukav** [comment]
Hi,
It seems like the LIPID17 results are in different units. May I ask what the unit of y-axis in this figure is? According to that, I'll update the files.

**Comment by ohsOllila** [comment]
The units should be nm<sup>-3</sup> in number density. These units are already used in MacRog and Berger results in the figure.

**Comment by batukav** [comment]
Hi,

The number of ions in Lipid17 simulations of POPC:POPS (5:1) are:

500mM: 52 (n_water=5760)
1000mM: 104 (n_water=5760)
2000mM: 208 (n_water=5760)
3000mM: 311 (n_water=5760)
4000mM: 415 (n_water=5760)

I also updated the ion number densities with the units nm^(-3)

**Comment by ohsOllila** [comment]
@batukav do you have the same amount of cations in simulations with calcium, sodium and potassium, i,e,, do the values above apply to all of these?

**Comment by ohsOllila** [comment]
@batukav now I realized that you had updated the information into the Zenodo repositories so the above question is obsolete.

**Comment by ohsOllila** [comment]
I have now updated the figure and tables. I realized that the Lipid14/17 simulations with calcium by @batukav were not listed in the table. After adding these, the table became too large and I separated the systems with calcium and monovalent ions to separate tables and put the latter in SI because those results are only in SI currently. 

The details of  Lipid14/17 simulations with calcium by @batukav are still missing from the SI and delivering details from some CHARMM36 simulations is still in progress. However, I close this issue because the original topics listed here are now solved.
